staging/src/k8s.io/apiserver/pkg/server/options/authentication.go

```diff
@@ -128,15 +135,28 @@ func (s *DelegatingAuthenticationOptions) AddFlags(fs *pflag.FlagSet) {
 
 	s.ClientCert.AddFlags(fs)
 	s.RequestHeader.AddFlags(fs)
+
+	fs.BoolVar(&s.SkipInClusterLookup, "authentication-skip-lookup", s.SkipInClusterLookup, ""+
+		"If false, the authentication-kubeconfig will be used to lookup missing authentication "+
+		"configuration from the cluster.")
+
 }
 
 func (s *DelegatingAuthenticationOptions) ApplyTo(c *server.Config) error {
-	var err error
-	c, err = c.ApplyClientCert(s.ClientCert.ClientCA)
+	clientCA, err := s.getClientCA()
+	if err != nil {
+		return err
+	}
+	c, err = c.ApplyClientCert(clientCA.ClientCA)
```

Comment:
No need to change it for this PR, but this function would be better named ApplyClientCA